fast_align_audio/_faa.c

```diff
@@ -2,6 +2,11 @@
 #include <math.h>
 #include <string.h>
 
+#ifdef _MSC_VER
```

Comment:
I would typically use _WIN32 instead since you're not checking against the version of the Microsoft compiler, but if it works it works.

ssize_t is not part of the standard library to my knowledge, but if you need a signed size variable this solution should be fine. I guess you have found it somewhere :-)

Reply:
Ok, thanks. It seems to me that just using `int` instead of `ssize_t` is a less painful solution